webcompat/views.py

```diff
@@ -349,7 +350,11 @@ def show_issue(number):
     if session.get('show_thanks'):
         flash(number, 'thanks')
         session.pop('show_thanks')
-    return render_template('issue.html', number=number)
+    # right now this is a double request... can we prevent
+    # that from happening? add in a caching layer? (maybe it already happens)
+    issue_data = proxy_issue(number)
```

Comment:
I think I can prevent the client from hitting our API endpoint entirely for an Issue (except for fetching labels, but we should be able to do that via the server eventually I think).

Reply:
Probably we should try https://github.com/webcompat/webcompat.com/blob/master/webcompat/helpers.py#L498 to not break HTTP caching.